Scripts/developer_scripts/cgal_create_release_with_cmake.cmake

```diff
@@ -70,7 +70,16 @@ foreach(pkg ${files})
         if ("${fext}" STREQUAL ".h" OR "${fext}" STREQUAL ".hpp")
           file(READ "${pkg_dir}/${f}" file_content)
           string(REPLACE "$URL$" "$URL: ${GITHUB_PREFIX}/${pkg}/${f} $" file_content ${file_content})
-          string(REPLACE "$Id$" "This file is from the release ${CGAL_VERSION} of CGAL" file_content ${file_content})
+          if(EXISTS ${CMAKE_BINARY_DIR}/.git)
```

Comment:
`CMAKE_SOURCE_DIR`

Reply:
cmake must be called at the root of the repository which is `CMAKE_BINARY_DIR`, so it should be correct. Note that I used only `CMAKE_BINARY_DIR` in this script.

Reply:
Here is something I did not know:
> When run in `-P` script mode, CMake sets the variables `CMAKE_BINARY_DIR`, `CMAKE_SOURCE_DIR`, `CMAKE_CURRENT_BINARY_DIR` and `CMAKE_CURRENT_SOURCE_DIR` to the current working directory.

https://cmake.org/cmake/help/v3.6/variable/CMAKE_BINARY_DIR.html

So `CMAKE_SOURCE_DIR` and `CMAKE_BINARY_DIR` are identical and set to the current working directory.